src/platform/utilities/feature-toggles/flipper-client.js

```diff
@@ -18,7 +18,9 @@ function FlipperClient({
   const _subscriberCallbacks = [];
 
   const _fetchToggleValues = async () => {
-    const response = await fetch(`${host}${toggleValuesPath}`);
+    const response = await fetch(`${host}${toggleValuesPath}`, {
+      credentials: 'include',
```

Comment:
TIL you have to tell fetch to pass cookies

Reply:
https://developers.google.com/web/updates/2015/03/introduction-to-fetch#sending_credentials_with_a_fetch_request